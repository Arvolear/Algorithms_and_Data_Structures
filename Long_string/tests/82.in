505462437311759803712821767518980446880533920
33304789187838948982880035547650981561479