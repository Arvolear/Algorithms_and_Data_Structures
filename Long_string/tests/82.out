505495742100947642661804647554528097862095399
505429132522571964763838887483432795898972441
16834319917039420456302793472914413921841939574433484094166011167019613425810824867680
15176